### DISCUSSION TITLE
UI: reordering the columns in the account's transactions view

### DISCUSSION THREAD

**Discussion by splisp** [moneymanagerex/moneymanagerex#6794] [Q&A] [answered]
I cannot have the order of the columns stay as I want. I don't mean the sort direction for the current sorted column, but the order of the columns: I drag the headers so that from left to right I have

Date - Status - Deposit - Withdrawal - Balance - Payee - Category - Notes

but next time I reopen the program the order has reverted to

Date - Payee - Status - Category - Withdrawal - Deposit - Balance - Notes

Windows 10 Professional, MoneyManagerEx Version 1.8.0 64-bit

**Comment by renato-mmex** [comment, marked as answer]
https://github.com/moneymanagerex/moneymanagerex/issues/4160

**Reply by Macaulay1234** [reply]
Yes
